README.markdown

```diff
@@ -247,10 +246,26 @@ Book.import books, recursive: true
 
 ### Options
 
+Key                     | Options               | Default            | Description
```

Comment:
I did my best scanning through the comments in the code to pick up all the options but double check me to make sure I didn't miss any options!

Reply:
Looks good!